openassessment/xblock/static/sass/oa/utilities/_shame.scss

```diff
@@ -4,6 +4,7 @@
 // NOTES: use for any bad-form/orphaned scss that knowingly violate edX FED architecture/standards (see - http://csswizardry.com/2013/04/shame-css)
 
 .openassessment {
+  @include direction();
```

Comment:
Just to make it clear, I'm not sure why the CSS direction isn't being inherited from the lms styles. But this fixed it (see the screenshots above in https://github.com/edx/edx-ora2/pull/664#issue-55488859).